accname/name/comp_hidden_not_referenced.html

```diff
@@ -13,16 +14,145 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_hidden_not_referenced">#comp_hidden_not_referenced</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<h2 class="ex" data-expectedlabel="heading label" data-testname="heading with interior hidden node">
-  heading
-  <span hidden>bogus</span>
-  label
+<button
+  class="ex"
+  data-expectedlabel="visible to all users"
+  data-testname="button containing a rendered, unreferenced element that is aria-hidden=true, an unreferenced element with the hidden host language attribute, and an unreferenced element that is unconditionally rendered"
+>
+  <span aria-hidden="true">hidden,</span>
+  <span hidden>hidden from all users,</span>
+  <span>visible to all users</span>
+</button>
+
+<button
+  class="ex"
+  data-expectedlabel="hidden but referenced,"
+  data-testname="button labelled by element that is aria-hidden=true"
+  aria-labelledby="button-label-2"
+>
+  <span aria-hidden="true" id="button-label-2">hidden but referenced,</span>
+  <span hidden>hidden from all users,</span>
+  <span>visible to all users</span>
+</button>
+
+<button
+  class="ex"
+  data-expectedlabel="hidden from all users but referenced,"
+  data-testname="button labelled by element with the hidden host language attribute"
+  aria-labelledby="button-label-3"
+>
+  <span aria-hidden="true">hidden,</span>
+  <span hidden id="button-label-3">hidden from all users but referenced,</span>
+  <span>visible to all users</span>
+</button>
+
+<!-- TODO: Restore the following test once expectation for `aria-hidden="false"`
+<!--       behavior is documented. See: -->
+<!--       https://github.com/w3c/aria/issues/1256 -->
```

Comment:
I know you worked hard on these, but I think it would be better to check in the patch without the commented code, and only add specific examples once the new issue came to resolution... Otherwise someone might see 1256 is resolved (in the future) and uncomment these even if they don't align with the resolution… Or also bad, these could just sit around commented indefinitely.

Reply:
maybe just keep your test names, since they should be clear enough to show what needs to be tested? such as:

```html
<!-- 
  Todo: once https://github.com/w3c/aria/issues/1256 resolved.

  - button labelled by an element that is aria-hidden=true which contains a nested child that is aria-hidden=false 
  - test name 2
  - test name 3
  - etc.
-->
```

Reply:
thanks for working so hard on these!

Reply:
Acknowledging that **August James** asked you to comment them, and **October James** told you to remove them, so I won't stop you from squash merging as-is... Take your pick. Thanks.

Reply:
Ha ha, no trouble at all, @cookiecrook  — happy for the opportunity to contribute at all. I went ahead and removed the test cases that are blocked by [aria#1256](https://github.com/w3c/aria/issues/1256) but left a comment identifying each one by name. 👍🏻 

Based on our quick conversation in the WPT meeting, I also left the expectation as is for the `...visibility:hidden with nested content that is visibility:visible` test case, so Firefox is currently the only pass for that one.